_examples/how-to-create-a-flutter-app-with-serverless.md

```diff
@@ -0,0 +1,408 @@
+});
+
+// Allow the API to access the table
+api.attachPermissions([table]);
+
+// Show the API endpoint in the output
+this.addOutputs({
+  ApiEndpoint: api.url,
+});
+```
+
+We are using the SST [`Api`](https://docs.serverless-stack.com/constructs/Api) construct to create our API. It simply has one endpoint (the root). When we make a `POST` request to this endpoint the Lambda function called `main` in `src/lambda.js` will get invoked.
+
+We also pass in the name of our DynamoDB table to our API as an environment variable called `tableName`. And we allow our API to access (read and write) the table instance we just created.
+
+### Reading from our table
+
+Our API is powered by a Lambda function. In the function we'll read from our DynamoDB table.
+
+{%change%} Replace `src/lambda.js` with the following.
+
+```js
+import AWS from "aws-sdk";
+
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+flutterrt async function main() {
+  const getParams = {
+    // Get the table name from the environment variable
+    TableName: process.env.tableName,
+    // Get the row where the counter is called "clicks"
+    Key: {
+      counter: "clicks",
+    },
+  };
+  const results = await dynamoDb.get(getParams).promise();
+
+  // If there is a row, then get the value of the
+  // column called "tally"
+  let count = results.Item ? results.Item.tally : 0;
+
+  return {
+    statusCode: 200,
+    body: count,
+  };
+}
+```
+
+We make a `get` call to our DynamoDB table and get the value of a row where the `counter` column has the value `clicks`. Since we haven't written to this column yet, we are going to just return `0`.
+
+{%change%} Let's install the `aws-sdk`.
+
+```bash
+$ npm install aws-sdk
+```
+
+And let's test what we have so far.
+
+## Starting your dev environment
+
+{%change%} SST features a [Live Lambda Development](https://docs.serverless-stack.com/live-lambda-development) environment that allows you to work on your serverless apps live.
+
+```bash
+$ npx sst start
+```
+
+The first time you run this command it'll take a couple of minutes to deploy your app and a debug stack to power the Live Lambda Development environment.
+
+```
+===============
+ Deploying app
+===============
+
+Preparing your SST app
+Transpiling source
+Linting source
+Deploying stacks
+dev-flutter-app-my-stack: deploying...
+
+ ✅  dev-flutter-app-my-stack
+
+
+Stack dev-flutter-app-my-stack
+  Status: deployed
+  Outputs:
+    ApiEndpoint: https://sez1p3dsia.execute-api.ap-south-1.amazonaws.com
+```
+
+The `ApiEndpoint` is the API we just created.
+
+Let's test our endpoint. Run the following in your terminal.
+
+```bash
+$ curl -X POST https://sez1p3dsia.execute-api.ap-south-1.amazonaws.com
+```
+
+You should see a `0` printed out.
+
+## Setting up our flutter app
+
+We are now ready to use the API we just created. Let's use [flutter-cli](https://flutter.dev/docs/get-started/install) to setup our flutter app.
+
+{%change%} Run the following in the project root.
+
```

Comment:
Add a step for installing the flutter cli.

Reply:
flutter doesn't use node, need to be installed manually.